toxav/msi.c

```diff
@@ -9,12 +9,22 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include "toxav_hacks.h"
+
 #include "../toxcore/ccompat.h"
 #include "../toxcore/logger.h"
+#include "../toxcore/net_crypto.h"
+#include "../toxcore/tox.h"
+#include "../toxcore/tox_private.h"
 #include "../toxcore/util.h"
 
 #define MSI_MAXMSG_SIZE 256
 
+#ifndef TOXAV_DEFINED
+#define TOXAV_DEFINED
+typedef struct ToxAV ToxAV;
+#endif /* TOXAV_DEFINED */
```

Comment:
we are including `toxav_hacks.h` which provides functions that already need this, so it is safe to assume this already exists here.